rust-runtime/aws-smithy-http-server-python/examples/stubgen.py

```diff
@@ -0,0 +1,391 @@
+#  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Is the `example` folder the right place for this file? We need to be able to vend it to customers somehow, right?

Reply:
Yes we need to vend it to customers somehow, but I'm not sure how yet. I think we'll need a build system eventually but currently the closest thing we have is the `Makefile` in the `examples` folder, so that's why I've put this file here.

Reply:
I think it is fine for this PR as we are just adding the functionality. We need to start thinking about how to vend this file. We should probably cut an issue about it.